src/flows/context/shared.tsx

```diff
@@ -0,0 +1,45 @@
+import React, {FC, useEffect, useState} from 'react'
+import {FlowContext} from 'src/flows/context/flow.current'
+import {useParams} from 'react-router'
+import {DEFAULT_PROJECT_NAME} from 'src/flows'
+import PageSpinner from 'src/perf/components/PageSpinner'
+
+import {hydrate} from 'src/flows/context/flow.list'
+import {RemoteDataState} from 'src/types'
+
+export const FlowProvider: FC = ({children}) => {
+  const [flow, setFlow] = useState(null)
+  const [loading, setLoading] = useState(RemoteDataState.NotStarted)
+  const {accessID} = useParams<{accessID: string}>()
+
+  useEffect(() => {
+    setLoading(RemoteDataState.Loading)
+    fetch(`${window.location.origin}/api/share/${accessID}`)
```

Comment:
i dont remember why we made this origin distinction and not just use API_PREFIX like everything else

Reply:
prolly for to change

Reply:
worked fine without it - so I removed it